components/hubspot/hubspot.app.mjs

```diff
@@ -0,0 +1,311 @@
+import axios from "axios";
+import hubspotSDK from "@hubspot/api-client";
+
+export default {
+  type: "app",
+  app: "hubspot",
+  propDefinitions: {
+    lists: {
+      type: "string[]",
+      label: "Lists",
+      description: "Select the lists to watch for new contacts.",
+      async options({ page }) {
+        const count = 250;
+        const offset = page * count;
+        const params = {
+          count,
+          offset,
+        };
+        const results = await this.getLists(params);
+        const options = results.map((result) => {
+          const {
+            name: label,
+            listId,
+          } = result;
+          return {
+            label,
+            value: {
+              label,
+              value: listId,
+            },
```

Comment:
Object values for "string" props aren't [compatible with the upcoming UI refresh](https://github.com/PipedreamHQ/pipedream/issues/2073). To support access to this prop's label, add the `withLabel` property.

```suggestion
    lists: {
      type: "string[]",
      label: "Lists",
      description: "Select the lists to watch for new contacts.",
      withLabel: true,
      async options({ page }) {
        const count = 250;
        const offset = page * count;
        const params = {
          count,
          offset,
        };
        const results = await this.getLists(params);
        const options = results.map((result) => {
          const {
            name: label,
            listId,
          } = result;
          return {
            label,
            value: listId,
```

Reply:
Resolved in https://github.com/PipedreamHQ/pipedream/pull/1419/commits/a9b45d8be3b84504dc22bb2748d925f0d5c1541f ✅